patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
+    for key, value in kwargs.iteritems():
+        get_params[key] = value
+    get_response = requests.get(patent_url, params = kwargs)
```

Comment:
Kind of what I was talking about before - there doesn't seem to be a reason to pass kwargs in here - can you elaborate on why you did this?